package.json

```diff
@@ -12,7 +12,7 @@
   },
   "author": "Onion Product Team <webtech@theonion.com>",
   "license": "UNLICENSED",
-  "devDependencies": {
+  "dependencies": {
```

Comment:
Shifted all these to dependencies from devDepedencies for moneyhole.

Moneyhole builds it's own webpack and directly imports from bulbs-elements source. Therefore it needs the same webpack plugins etc.

We should be smarter about how we do this.